hr_employee_id/readme/INSTALL.rst

```diff
@@ -0,0 +1,7 @@
+To install this module, you need to:
+
+* clone the branch 11.0 of the repository https://github.com/OCA/hr
```

Comment:
Outdated. It is version 16. To avoid this kind of outdate problems with next versions, I would use 'Clone the main branch' or 'clone the branch you need' or even nothing related to the branch.